onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1320,7 +1320,15 @@ CUDAExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
       defs_outside_cuda.insert(node.OutputDefs().cbegin(), node.OutputDefs().cend());
       continue;
     }
-    cuda_kernel_def = GetKernelRegistry()->TryFindKernel(node, Type());
+
+    for (auto registry : kernel_registries) {
```

Comment:
The CUDA kernel registry itself should already be part of the `kernel_registries` passed to the `GetCapability()` method